TEveEventDisplay/src/TEveMu2e_base_classes/TEveMu2eMCInterface.h

```diff
@@ -40,6 +40,9 @@ namespace mu2e{
         TEveElementList *fTrackList2DXZ;
         TEveElementList *fTrackList3D;
         std::vector<int> particleIds_;
+        constexpr Int_t linewidth(){ return 3; }
+        constexpr Int_t fontsize(){ return 15; }
+        constexpr double text_z_pos(){ return 3000.; }
```

Comment:
I recommend doing these as variables not as functions.  For example

constepxpr Int_t linewidth = 3;

Reply:
You might also consider creating a "style" class that has default values - that would allow you to treat the style options as a unit and should reduce repetition and clutter in the code.  Whether or not this really makes sense depends on details of how it will be used.  Please consider this idea as you move forward.

Reply:
yes, i thought the same thing actually